src/components/completing-party/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/completing-party",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
This gets done automatically when by `lerna version --include-merged-tags`. Refer back to the readme for this project. Please revert this change.

Reply:
OK, since you've merged this already, I will fix it when I run steps 14-15 in the readme.

Reply:
actually, @lewischenstudio can revert it in his upcoming PR to update the middle name length

Reply:
I can't update it without pushing another commit, so I just went with the next version number.

Version 2.0.5 is now ready for import into other UIs.